modules/tinymce/src/core/test/ts/browser/init/InitIframeAriaTextTest.ts

```diff
@@ -7,30 +7,44 @@ import { assert } from 'chai';
 import Editor from 'tinymce/core/api/Editor';
 
 describe('browser.tinymce.core.init.InitIframeAriaTextTest', () => {
-  const defaultIframeTitle = 'Rich Text Area. Press ALT-0 for help.';
-  const customIframeTitle = 'Cupidatat magna aliquip.';
+  const defaultIframeTitle = 'Rich Text Area';
+  const defaultIframeAriaText = 'Rich Text Area';
+  const defaultIframeAriaTextWithHelpPlugin = 'Rich Text Area. Press ALT-0 for help.';
```

Comment:
Same as above, I would suggest building the strings instead of assigning completely new different strings.